OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -118,13 +118,18 @@ public Order IssueOrder(Actor self, IOrderTargeter order, in Target target, bool
 
 		public void ResolveOrder(Actor self, Order order)
 		{
-			if (order.OrderString != OrderID)
-				return;
-
-			if (!order.Queued)
-				Path.Clear();
-
-			Path.Add(self.World.Map.CellContaining(order.Target.CenterPosition));
+			switch (order.OrderString)
+			{
+				case "Stop":
+					Path.Clear();
+					break;
+				case OrderID:
+					if (!order.Queued)
+						Path.Clear();
+
+					Path.Add(self.World.Map.CellContaining(order.Target.CenterPosition));
+					break;
+			}
```

Comment:
In this case a switch just makes it less readable, please use regular if conditional statements

Reply:
Will do! What do you prefer: Two single if-Statements or one if/else if Statement?

Reply:
whatever is more readable / makes more sense

Reply:
After trying the if-Statements I'm a little bit surprised, but (of course!) you are correct. It's more readable that way!
I've opted for the early return approach instead of two if-Statements. What do you think?